### PR TITLE
Fix and Update Basic Usage's and Core page

### PR DESCRIPTION
**Pull request by vairodp**
# Description

Fix the Basic Usage page docs, by updating the code, fixing it, and also fixing the grammar.
In the second commit I fixed also the API -> Core page, by removing deprecated docs, fixing grammar and inconsistencies 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by pseudo-rnd-thoughts** [comment]
Amazing, I will read through all of the changes tomorrow evening. 
Are you going to review any more files for this PR?

**Comment by vairodp** [comment]
> Amazing, I will read through all of the changes tomorrow evening. Are you going to review any more files for this PR?

Not in this PR, my plan is to do a PR for every "section" of work that I'm doing and I've already done another PR for the tutorial section. 
If you think is a bad idea I can continue my work in this PR

File: gymnasium/core.py [hunk 1]
```diff
@@ -353,7 +353,7 @@ class ObservationWrapper(Wrapper):
     ``observation["target_position"] - observation["agent_position"]``. For this, you could implement an
     observation wrapper like this::
 
-        class RelativePosition(gymnasium.ObservationWrapper):
+        class RelativePosition(gymn.ObservationWrapper):
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/core.py:356]: gym

**Review by pseudo-rnd-thoughts — CHANGES_REQUESTED** [review]
This looks good except for the `gymn` to `gym`

File: gymnasium/core.py (hunk 1 above)
Comment (vairodp) [reply gymnasium/core.py]: Aahahahah true, fixed

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**